modules/ROOT/pages/kubernetes/quickstart-analytics-cluster.adoc

```diff
@@ -0,0 +1,168 @@
+:description: How to deploy a Neo4j cluster to a cloud or a local Kubernetes cluster using Neo4j Helm chart.
+[role=enterprise-edition]
+[[quick-start-analytic-cluster]]
+= Quickstart: Deploy a Neo4j cluster for analytic queries
+
+_The feature is available in the Neo4j Helm chart from version 5.14._
+
+This quickstart shows how to configure and deploy a special Neo4j cluster without fault tolerance to support analytic queries.
+Information on using Neo4j’s graph data science library in a cluster can be found in the link:https://neo4j.com/docs/graph-data-science/current/[Neo4j Graph Data Science library documentation].
+
+The cluster is deployed to a cloud or a local Kubernetes cluster using the Neo4j Helm chart.
+Because the GDS library does not support fault tolerance, you can deploy a cluster with one primary server for the transaction workload and an N number of secondary servers for the GDS analytics.
```

Comment:
GDS library does not support fault tolerance ? are you sure ? i don't think so , you can install GDS in a cluster of 5 members where we have 3 primary and 2 secondary this makes it fault tolerant.

So , i guess this statement needs to be removed.

Reply:
If that's not true, then the docs are not correct:
"The analytics cluster can be set up in two ways, with fault tolerance or without fault tolerance. Bear in mind that the GDS library does not support fault tolerance and therefore GDS should only be deployed on servers configured for analytic queries, as described below."